analyses/add_leaflet_maps.qmd

```diff
@@ -0,0 +1,226 @@
+### Largest 100 Values
+
+Be careful interpreting values which are the max and min of the raw value, since ties are not accounted for. For example, if there are 10,000 parcels which are 0 feet from a newly constructed building, the map will not be a full representation.
+
+```{r}
+highest_100 <- leaflet_data %>%
+  arrange(desc(!!sym(target_feature_value))) %>%
+  dplyr::slice(1:100)
+
+create_leaflet_map(highest_100, {{ target_feature_value }}, "Largest 100 Values")
+```
+
+### Lowest 100 Values
+
+Be careful interpreting values which are the max and min of the raw value, since ties are not accounted for. For example, if there are 10,000 parcels which are 0 feet from a newly constructed building, the map will not be a full representation.
+
+```{r}
+lowest_100 <- leaflet_data %>%
+  distinct(meta_pin, .keep_all = TRUE) %>%
+  arrange(!!sym({{ target_feature_value }})) %>%
+  slice(1:100)
+
+create_leaflet_map(lowest_100, {{ target_feature_value }}, "Lowest 100 Values", order_scheme = "low")
+```
+
+### Highest 100 SHAP Values
+
+```{r}
+highest_100 <- leaflet_data %>%
+  arrange(desc(shap_total)) %>%
+  slice(1:100)
+
+create_leaflet_map(highest_100, "shap_total", "Highest 100 SHAPs")
+```
+
+### Lowest 100 SHAP Values
+
+```{r}
+lowest_100 <- leaflet_data %>%
+  arrange(shap_total) %>%
+  slice(1:100)
+
+create_leaflet_map(lowest_100, "shap_total", "Lowest 100 SHAPs", order_scheme = "low")
+```
+:::
+
+## Largest FMV Changes
+
+Multicard parcels have heuristic which limits their change. The added feature may trigger (or not trigger it), leading to changes much larger than the added feature's impact.
+
+::: panel-tabset
+### 100 Largest FMV Increases
+
+```{r}
+largest_fmv_increases <- leaflet_data %>%
+  arrange(desc(diff_pred_pin_final_fmv)) %>%
+  slice(1:100)
+
+# Call the function with the pre-sliced dataset
+create_leaflet_map(largest_fmv_increases, "diff_pred_pin_final_fmv", "Largest FMV Increases (%)")
+```
+
+### 100 Largest FMV Decreases
+
+Multicard parcels have heuristic which limits their change. The added feature may trigger (or not trigger it), leading to changes much larger than the added feature's impact.
+
+```{r}
+largest_fmv_decreases <- leaflet_data %>%
+  arrange(diff_pred_pin_final_fmv) %>%
+  slice(1:100)
+
+create_leaflet_map(largest_fmv_decreases, "diff_pred_pin_final_fmv", "Largest FMV Decreases (%)", order_scheme = "low")
+```
+
+### 100 Largest FMV Initial Increases
+
+```{r}
+largest_fmv_increases <- leaflet_data %>%
+  arrange(desc(diff_pred_pin_initial_fmv)) %>%
+  slice(1:100)
+
+# Call the function with the pre-sliced dataset
+create_leaflet_map(largest_fmv_increases, "diff_pred_pin_initial_fmv", "Largest FMV Increases (%)")
+```
+
+### 100 Largest Initial FMV Decreases
+
+```{r}
+largest_fmv_decreases <- leaflet_data %>%
+  arrange(diff_pred_pin_initial_fmv) %>%
+  slice(1:100)
+
+create_leaflet_map(largest_fmv_decreases, "diff_pred_pin_initial_fmv", "Largest FMV Decreases (%)", order_scheme = "low")
+```
+
```

Comment:
We have a bunch of FMV increases. We should probably only have a couple, but just let me know which ones you think are best to keep. There are some pretty large swings due to the multicard heuristics, which we may or may not want to track.